UI/Pages/SearchResults.cshtml

```diff
@@ -130,45 +158,77 @@
+                                <a href="@tpUrl" class="govuk-link">@item.Name</a>
+                            </h2>
+                            <govuk-summary-list>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    <govuk-summary-list-row-key>
+                                        Subjects covered
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value>
+                                        <ul class="govuk-list govuk-body-s govuk-list-bullets-mobile-view" data-testid="results-subjects">
+                                            @foreach (var keyStageSubjects in 
+                                                item.SubjectsCoverage.Select(e => e.Subject).Distinct().OrderBy(e => e.KeyStageId)
+                                                    .OrderBy(e => e.Name).GroupBy(e => e.KeyStageId))
+                                            {
+                                                <li>@(((KeyStage) keyStageSubjects.Key).DisplayName()) - @keyStageSubjects.DisplayList()</li>
+                                            }
+                                        </ul>
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    <govuk-summary-list-row-key>
+                                        Type of tuition
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value data-testid="type-of-tuition">
+                                        @string.Join(", ", item.TuitionTypes.Select(e => e.Name))
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    <govuk-summary-list-row-key>
+                                        Tuition partner information
+                                    </govuk-summary-list-row-key>
+                                    <govuk-summary-list-row-value data-testid="results-description">
+                                        @item.Description
+                                    </govuk-summary-list-row-value>
+                                </govuk-summary-list-row>
+                                <govuk-summary-list-row class="govuk-body-s">
+                                    @{
+                                        var shortlist = Model.SelectableTuitionPartners
+                                            .First(tp => tp.SeoUrl == item.SeoUrl);
+                                    }
+                                    <govuk-summary-list-row-key>
+                                        Add to my shortlist
```

Comment:
This should be the `<label>` for the checkbox

Reply:
Yes, it should be. The challenge is, based on the 'dgs' system checkboxes, it expects a label in the same div as the checkbox. And when using that with the `govuk-summary-list-row` element it gets even more interesting.